core/src/main/java/dev/keva/core/command/mapping/CommandMapper.java

```diff
@@ -115,7 +116,7 @@ public void init() {
                         } catch (Exception e) {
                             log.error(e.getMessage(), e);
                             if (e instanceof InvocationTargetException) {
-                                if (e.getCause() instanceof ClassCastException) {
+                                if (e.getCause() instanceof SerializationException || e.getCause() instanceof ClassCastException) {
```

Comment:
Should we wrap the `SerializationException` thrown during deserilise to a custom one, so that we can distinguish it from the same thrown in call to serialise?

Reply:
Like we need to merge SerializationException and ClassCastException to something like `KevaTypeException` right?

Reply:
I meant we have code like this:
```
try {
            byte[] value = chronicleMap.get(key);
            LinkedList<BytesValue> list;
            list = value == null ? new LinkedList<>() : (LinkedList<BytesValue>) SerializationUtils.deserialize(value); <- can throw
            for (byte[] v : values) {
                list.addFirst(new BytesValue(v));
            }
            chronicleMap.put(key, SerializationUtils.serialize(list)); <- can throw
            return list.size();
        } finally {
            lock.unlock();
        }
```      
From 2 places same exception can be thrown but only 1 of them indicates client error.

Reply:
SOunds good!